security-demo/release/app-binaries/conf/edge.yaml

```diff
@@ -0,0 +1,35 @@
+mqtt:
```

Comment:
Can we reuse the config file of edge-core?

Reply:
yes, once there is a release script, all the configs will be generated automatically.

Reply:
I suggest to take this into consideration as part of script optimizations. For the initial version, there are too many script functions.